integration/kubernetes/k8s-number-cpus.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
```

Comment:
A niggle - in the commit message body, rather than referencing github Issue numbers, can we have a textual explanation please - it makes viewing git logs and history on the command line a lot more meaninful and easier (as one does not have to navigate off to the web/github to look for the finer details).
thanks.